### ISSUE TITLE
excel file indices missing - can't download specific books

### ISSUE THREAD

**Issue by AntWarland** [alexgand/springer_free_books#73]
I want to download the epubs for a list of 38 books but there's no indices and row number has no relation to book downloaded.. please help

**Comment by chaosAD** [comment]
The indices are on the `A` column. The column is immediately to the left of the `Book Title`.

**Comment by AntWarland** [comment]
The book titles are on the A column for me (using this link):
https://resource-cms.springernature.com/springer-cms/rest/v1/content/17858272/data/v4

**Comment by chaosAD** [comment]
![indices](https://user-images.githubusercontent.com/2726488/80720763-f9451780-8b2f-11ea-9667-1195b060f39f.png)

**Comment by AntWarland** [comment]
I've tried downloading the file again and that column is still not there....

**Comment by chaosAD** [comment]
You are right, the original Excel file do not have column with the indices I was referring to. Try this (I know it is crude):
```bash
python main.py -i 1
``` 
Then wait till it finishes. Go to the `./download` sub-folder and open `table_v4.xlsx` file.

**Comment by AntWarland** [comment]
Okay, they're in that one! Thanks :)

**Comment by chaosAD** [comment]
To avoid confusion, I have amended the script to refer to the real Excel index instead of the newly injected index (red). This is fixed in PR #74.

So now, the first book, which is `Fundamental of Power Electronics` is on index 2 instead of 0. To download only that book
```bash
python main.py -i 2
```

**Comment by cszesther** [comment]
Hi! Index 383 `Essentials of Business Analytics` should be index 382
